muxers/yamux/src/lib.rs

```diff
@@ -39,15 +39,7 @@ use std::{
 use thiserror::Error;
 
 /// A Yamux connection.
-pub struct Yamux<S>(Mutex<Inner<S>>);
```

Comment:
Neat!

Reply:
Well, looking at the current build failures this may have been a mistake :D